docs/topologies/star-shaped.md

```diff
@@ -0,0 +1,53 @@
+# Star-Shaped Head Network
+
+:hammer_and_wrench: This document is a work in progress
+
+This document details the behaviour of so-called _Star-shaped Hydra Network_.
+
+## Summary
+
+A _Star-shaped Hydra Network_ or more precisely a Star-shaped Heads Network is comprised of:
+
+* A central _Server_ node providing "Head-as-a-service", with low expected downtime, probably operated by some company or organisation with enough resources to host this service,
+* _Client_ nodes, either DApp instances, or mobile/personal wallets, which might not be always online and possibly can come and go.
+
+![Star-shaped Heads Network](./star-shaped-general.jpg)
+
+Client nodes want to be able to interact with each other efficiently, at a low cost, using L2 solution, with all the Hydra safety guarantees, but without bearing the operational burden of operating an always online "full" Hydra node (eg. using an embedded version of the node, or a lighweight version). There might be a lot of them, say in the 100s or even 1000s but they are not always all live and up at the same time.
+
+Client nodes establish pairwise Heads (eg. _channels_) with the server: This setup is simpler than with a normal multiparty head because the server has as a well-known identity and the client can always provide the needed parameters (keys, IP) to the server when setting up the Head using some specific service whose definition is outside of the scope of this document.
+
+Transactions a client node posts in "its" Head should be _reflected_ by the server into the other Heads it maintain.
```

Comment:
The **should** is the interesting part here. How does a client enforce it?  ---> HTLC
But the explanation is somewhat missing from the document :thinking:  ?

Reply:
It is because I don't know how this should be done. I have vague ideas (HTLC) but this is a bit hand-wavy at the moment, I would be interested in insights from more knowledgeable people (you, for example), on how to provide this guarantee.